calico-vpp-agent/watchers/uplink_route_watcher.go

```diff
@@ -34,6 +43,22 @@ type RouteWatcher struct {
 	stop              bool
 	lock              sync.Mutex
 	closeLock         sync.Mutex
+	routeEventChan    chan common.CalicoVppEvent
+	FakeNextHopIP4    net.IP
+	FakeNextHopIP6    net.IP
+}
+
+func NewRouteWatcher() *RouteWatcher {
+	routeWatcher := &RouteWatcher{
+		routeEventChan: make(chan common.CalicoVppEvent, common.ChanSize),
+		FakeNextHopIP4: common.VppManagerInfo.FakeNextHopIP4,
+		FakeNextHopIP6: common.VppManagerInfo.FakeNextHopIP6,
```

Comment:
Maybe we could pass it as a parameter of `NewRouteWatcher()` (same as for the LinkWatcher), just to make it clear that we depend on those values being set in the caller, and that it needs to happen after `WaitForVppManager()` is called